### PR TITLE
Remove references to the Content API

### PR DESCRIPTION
**Pull request by alecgibson**
The [`content_api`](https://github.com/alphagov/govuk_content_api) has
been retired, and all content has been migrated to the
[`content_store`](https://github.com/alphagov/content-store).

Therefore, we can remove all references to the Content API, and rely
solely on the Content Store.

### Trello

https://trello.com/c/2BFMkQvB/124-stop-using-the-content-api-in-the-frontend-app

### CODE REVIEW

File: test/fixtures/child-tax-credit.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     },
     "title": "Get Child Tax Credit",
```
Comment (tijmenb) [inline test/fixtures/child-tax-credit.json]: This file looks like a content-api artefact, so we should be able to remove it too.

**Comment by fofr** [comment]
What were Publications in the context of frontend? I only recall us moving Whitehall Publications to government-frontend.

**Comment by davidbasalla** [comment]
@fofr I think `PublicationPresenter` in the context of Frontend was the base presenter which was used for all formats before Custom Team gave each format a dedicated presenter. So I think removing it should be fine, nothing should be using it anymore... The wording in the PR about Publications being rendered by government-frontend probably isn't 100% correct, the term Publication in Frontend has been a bit misleading...

**Comment by alecgibson** [comment]
@davidbasalla I didn't know that history thanks - I'll update the commit message.

**Comment by alecgibson** [comment]
@tijmenb I've removed that and other unused text fixtures.

**Comment by alecgibson** [comment]
@tijmenb / @fofr / @davidbasalla are you happy with this change?

File: README.md [hunk 2]
```diff
@@ -49,12 +47,10 @@ To run the application standalone, run
...
 
 ```
-PLEK_SERVICE_STATIC_URI=http://127.0.0.1:3013 PLEK_SERVICE_CONTENTAPI_URI=https://www.gov.uk/api ./startup.sh
+PLEK_SERVICE_STATIC_URI=http://127.0.0.1:3013 ./startup.sh
 ```
 
-which uses the production
-[content API](https://github.com/alphagov/govuk_content_api) and a local copy of
-static.
+which uses the production a local copy of static.
```
Comment (fofr) [inline README.md]: Copy here doesn't read correctly

**Comment by davidbasalla** [comment]
I'll have a look through!

**Comment by davidbasalla** [comment]
You can probably remove `require 'gds_api/content_api'` from:
https://github.com/alphagov/frontend/blob/master/app/controllers/application_controller.rb#L2
https://github.com/alphagov/frontend/blob/master/config/initializers/gds_api.rb#L2

**Comment by davidbasalla** [comment]
https://github.com/alphagov/frontend/blob/master/test/unit/helpers/application_helper_test.rb#L37 still refers to `artefact`. Artefacts were the old world concept for Documents within this app, so could be updated to 'ContentItem' probably

**Comment by davidbasalla** [comment]
Looks like you can remove `def error_404; error 404; end` in the ApplicationController as well, that's not being used anymore

**Comment by davidbasalla** [comment]
Looks good! Only a few minor comments ^

**Comment by alecgibson** [comment]
@fofr / @davidbasalla I've addressed your comments - could you please re-review?

**Pull request merged**